fetchApi.tsx

```diff
@@ -1,18 +1,19 @@
-import { Vehicle } from "./models";
+import { Vehicle, VehicleList } from "./types";
 
 const fetchAPI = {
```

Comment:
To jest trochę kwestia stylistyczna, więc możesz robić jak według Ciebie jest lepiej, ale...
możesz nie tworzyć obiektu `fetchApi` w tym pliku, tylko exportować każdą funkcje osobno:
`export async function getVehiclesListAndCount(....): ... {`

Dzięki temu wiesz od razu które funkcje są lokalne, a które eksportowalne. A używć możesz ich tak samo:
albo
`import fetchAPI from "../fetchApi";`
albo
`import { getVehiclesListAndCount } from "../fetchApi";`

(BTW, jak robisz `export default NAZWA`, to potem w imporcie możesz używać dowolnej innej nazwy, to nie ma żadnego znaczenia - JS nie wie jak nazywa się wyeksportowany obiekt)